Makefile

```diff
@@ -112,11 +112,9 @@ check:
 	@echo
 	@echo "Compiling in Debug mode..."
 ifeq ($(RUNTIME), mono)
-# Enabling EnforceCodeStyleInBuild and GenerateDocumentationFile as a workaround for some code style rules (in particular IDE0005) being bugged and not reporting warnings/errors otherwise.
-	@$(MSBUILD) -t:build -restore -p:Configuration=Debug -warnaserror -p:TargetPlatform=$(TARGETPLATFORM) -p:EnforceCodeStyleInBuild=true -p:GenerateDocumentationFile=true
+	@$(MSBUILD) -t:build -restore -p:Configuration=Debug -warnaserror -p:TargetPlatform=$(TARGETPLATFORM)
 else
-# Enabling EnforceCodeStyleInBuild and GenerateDocumentationFile as a workaround for some code style rules (in particular IDE0005) being bugged and not reporting warnings/errors otherwise.
-	@$(DOTNET) build -c Debug -nologo -warnaserror -p:TargetPlatform=$(TARGETPLATFORM) -p:EnforceCodeStyleInBuild=true -p:GenerateDocumentationFile=true
+	@$(DOTNET) build -c Debug -nologo -warnaserror -p:TargetPlatform=$(TARGETPLATFORM)
```

Comment:
Can you please push a test commit with a redundant `using` for me?

Also I get a `IDE0004 Cast is redundant` in `LZOCompression`, so the PR may benefit from a rebase onto `bleed` anyway.

Reply:
Did something change here or was this already broken on bleed?

Reply:
afaik we just merged a PR fixing `LZOCompression `

Reply:
#20746 fixed the cast.

Reply:
No I mean that we need to make clean. I simply commented on the wrong comment. 🤦‍♂️